static/templates/user_info_popover_content.handlebars

```diff
@@ -6,6 +6,8 @@
             {{#if is_active }}
                 {{#if is_bot}}
                 <i class="zulip-icon bot" aria-hidden="true"></i>
+                {{else if is_admin}}
+                <img class="crown" src="/static/images/crown.svg" />
```

Comment:
Instead of adding the source attribute, you should've added the .svg to the `static/assets/icons` folder to be automatically generated into a webfont icon.